src/utils/getErrors.js

```diff
@@ -0,0 +1,35 @@
+const cases = ['SNAKE', 'KEBAB', 'CAMEL', 'PASCAL', 'UPPER'];
+const regs = '"/<TEXT_TO_CONVERT>?toCase=<CASE_NAME>"';
+const errorMessages = [
+  {
+    message: `Text to convert is required. Correct request is: ${regs}.`,
+  },
+  {
+    message: `"toCase" query param is required. Correct request is: ${regs}.`,
+  },
+  {
+    message: `This case is not supported. Available cases: ${cases.join(
+      ', ',
+    )}.`,
+  },
+];
```

Comment:
make it like an object
It would be easy to take value from it
```suggestion
const errorMessages = {
    Error: `Text to convert is required. Correct request is: ${regs}.`,
    Error2: `"toCase" query param is required. Correct request is: ${regs}.`,
    Error3: `This case is not supported. Available cases: ${cases.join(', ',)}.`,
},;
```